examples/pylint/C0321_multiple_statements.py

```diff
@@ -0,0 +1,8 @@
+def pos(temp):
+    """Return "positive" if <temp> is greater than 0. ELse return "negative".
+    @type temp: int
+    @rtype: str
+    """
+    if temp > 0: return "positive"
```

Comment:
This is an example for the message multiple statements which is only emitted when I do this.